### DISCUSSION TITLE
lib/assets/images not found when running ubiquibot locally

### DISCUSSION THREAD

**Discussion by umershaikh123** [ubiquity/ubiquibot#650] [Q&A] [answered]
repo : https://github.com/ubiquity/ubiquibot

All the env is set .
![image](https://github.com/ubiquity/ubiquibot/assets/42178214/9130262c-22f9-428b-ae38-b80e578aca60)

```yarn start ``` is working but when running the bot it crashes at some point due do missing lib/assets/images 

![image](https://github.com/ubiquity/ubiquibot/assets/42178214/00eef3a3-a7a3-46fc-aa78-4d73e71cb37f)

**Comment by 0x4007** [comment]
I'm not sure specifically for your error case off hand but I know that you can remove references to any images and the bot will work fine. 

This was for a partially implemented feature, for weekly reviews on team performance on telegram. 

@rndquu @0xcodercrane what do you think we should do? Remove the image directory until the feature is re implemented?

**Reply by rndquu** [reply]
I also got this error some time ago but I don't remember how I fixed it. @wannacfuture @whilefoo help

**Reply by cuddle-dev** [reply]
Yes, that error comes from the weekly reviews feature on Telegram which has a couple of bugs/compatibility issues(ex; we should not use `fs` library as long as the bot is a serverless function). 

So I totally agree to remove the image directory for now, if the issue still persists from the telegram analytics feature, we can comment out that feature until we fix it.

**Reply by wannacfuture** [reply]
it is also happening on my side. I just disabled that feature as temporary solution.

**Reply by whilefoo** [reply]
What I did to fix it was:
```
mkdir -p lib/src/assets/images
cp src/assets/images/* lib/src/assets/images/
```

**Reply by web4er** [reply, marked as answer]
My recent PR removes the symlink script. The symlink script is not needed anymore because the build script is moving the original images. https://github.com/ubiquity/ubiquibot/pull/607/commits/d4967526b454512c9de436e5c3193ac4b97e2ad8
It was merged yesterday, so pull the recent changes and try.  This addresses the first part of the issue `yarn start:watch` .
